spec/features/user_signup_functionality_spec.rb

```diff
@@ -0,0 +1,100 @@
+require "rails_helper"
+
+describe 'a new user signin up', type: :feature do
+
+  describe "the user does not already exist" do
+
+    before(:each) do
+      visit "/"
+      click_link("Create Account")
+    end
+
+    it "can create a new account" do
+      fill_in('new_user[email]', with: 'Jeff@gmail.com')
+      fill_in('new_user[user_name]', with: 'Jeffy')
+      fill_in('new_user[password]', with: 'pass')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.